tools/accuracy_checker/accuracy_checker/main.py

```diff
@@ -196,94 +327,21 @@ def build_arguments_parser():
     parser.add_argument(
```

Comment:
it is OV related (myriad, HDDL are VPU devices)

Reply:
Moved